### DISCUSSION TITLE
NFC and Bluetooth sync

### DISCUSSION THREAD

**Discussion by MarjovanLier** [j-kaltes/Juggluco#190] [General]
Hi,

First of all, thank you for all the hard work and help you provide to the community. I've found the app and data helpful and informative.

We're currently using Juggluco and then sending the data to Nightscout. But I noticed it's only syncing the Bluetooth data and not the NFC data (as it looks to me) 

I'm a data person. I like to collect data over time and then do analysis on it and I think it would be helpful to sync somewhere. 

I'm not sure if I'm missing something. 

How can I make the NFC data sync to somewhere as well?

**Comment by j-kaltes** [comment]
You can export all data with [left middle menu->Export](https://www.juggluco.nl/Jugglucohelp/helpexport.html) or by turning on the Web server in Juggluco ([Left menu->Settings->Web server](https://www.juggluco.nl/Jugglucohelp/Nightscouthelp.html)) and going to https://www.juggluco.nl/Juggluco/webserver.html?urlstart=http://127.0.0.1:17580 on the same phone.
